src/gluonts/time_feature/_base.py

```diff
@@ -163,6 +163,31 @@ def week_of_year_index(index: pd.PeriodIndex) -> np.ndarray:
         week = index.week
     return week.astype(float).values - 1
 
+class fourier_time_features:
```

Comment:
I think you can keep this definition local to the `gluonts.ext.r_forecast` package for now. Also, we use the CamelCase convention for class names